.travis.yml

```diff
@@ -17,15 +16,18 @@ env:
 
 matrix:
   fast_finish: true
-  allow_failures:
-    - os: osx
   include:
-    - python: 2.7
+    - sudo: required
+      python: 2.7
       env: ENV=lint
       before_script: TOX_ENV=lint
-    - python: 2.7
+    - sudo: required
+      python: 2.7
       env: ENV=docs
       before_script: TOX_ENV=docs-travis
+      before_install:
+        - sudo apt-get update -qq
```

Comment:
I can't figure out why the addons stuff is not working (I tried many different things) :/

It seems that it works for all the tasks except the docs one (https://travis-ci.org/apache/libcloud/jobs/104312529).

I checked other projects and they use the same syntax and it works just fine for them.